src/harness/compilerRunner.ts

```diff
@@ -52,7 +52,7 @@ class CompilerBaselineRunner extends RunnerBase {
     private makeUnitName(name: string, root: string) {
         const path = ts.toPath(name, root, (fileName) => Harness.Compiler.getCanonicalFileName(fileName));
         const pathStart = ts.toPath(Harness.IO.getCurrentDirectory(), "", (fileName) => Harness.Compiler.getCanonicalFileName(fileName));
-        return path.replace(pathStart, "/");
+        return pathStart ? path.replace(pathStart, "/") : path;

```

Comment:
This fixes what was breaking web tests: if `pathStart` was `""` then `path.replace(pathStart, "/")` would put an extra "/" at the beginning (leading to `//src/...`). Then when the host was determining file existence it would look for strings starting with `/src`, which there were none of.